DataFormats/HcalRecHit/src/HcalRecHitDump_cfg.py

```diff
@@ -0,0 +1,50 @@
+# Run with something like the following:
```

Comment:
The problem was me putting the updated file (at my post-PR update) to "src" instead of "test". Old file removed, new file moved to correct location.